From 288cdb442b1405cf3f34e59203e481cc3f2bbc6d Mon Sep 17 00:00:00 2001
From: Ghennadi Procopciuc <ghennadi.procopciuc@nxp.com>
Date: Fri, 20 May 2022 08:54:55 +0300
Subject: [PATCH 19/30] configs: s32cc: Increase line buffer size

This allows easy handling of long commands as the ones
present in config_distro_bootcmd.h.

Issue: ALB-8903
Upstream-Status: Pending 

Signed-off-by: Ghennadi Procopciuc <ghennadi.procopciuc@nxp.com>
Signed-off-by: Zhantao Tang <zhantao.tang@windriver.com>
---
 include/configs/s32-cc.h | 1 +
 1 file changed, 1 insertion(+)

diff --git a/include/configs/s32-cc.h b/include/configs/s32-cc.h
index 0afd916075..5f6a71cc9f 100644
--- a/include/configs/s32-cc.h
+++ b/include/configs/s32-cc.h
@@ -334,4 +334,5 @@
 #  define FSL_QSPI_FLASH_SIZE		(SZ_64M)
 #endif
 
+#define CONFIG_SYS_CBSIZE		(SZ_512)
 #endif
-- 
2.17.1

